pytest_selenium/pytest_selenium.py

```diff
@@ -265,3 +265,12 @@ def pytest_addoption(parser):
                      metavar='str',
                      help='selenium eventlistener class, e.g. '
                           'package.module.EventListenerClassName.')
+    group._addoption('--host',
+                     metavar='str',
+                     help='host that selenium server is listening on. '
+                          '(default: %default)')
```

Comment:
The defaults are a little more obscured now. We could say something like: "host that the selenium server is listening on, which will default to the cloud provider default or localhost."